src/routes.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Route, BrowserRouter, Switch } from 'react-router-dom';
+import HomePage from './views/HomePage';
+import LoginPage from './views/LoginPage';
+import StartPage from './components/StartPage';
+import SignupPage from './views/SignupPage';
```

Comment:
All views should be imported in one line. See similar comment above.

Reply:
I have implemented the feedback. Thank you.